### PR TITLE
Four space indention and call-chain reorder

### PR DESCRIPTION
**Pull request by vanillajonathan**
Re-order AddCommandLine after AddJsonFile.

### CODE REVIEW

File: aspnetcore/fundamentals/hosting.md [hunk 1]
```diff
@@ -159,64 +159,64 @@ Use [Configuration](configuration.md) to set configuration values to be used by
...
-     .UseKestrel()
-     .UseStartup<Startup>()
-     .Start(urls.ToArray());
-
-   using (host)
-   {
-     Console.ReadLine();
-   }
-   ```
+    "http://*:5000",
```
Comment (GuardRex) [inline aspnetcore/fundamentals/hosting.md]: Add 4 spaces here and on the next line.
```
var urls = new List<string>() {
        "http://*:5000",
        "http://localhost:5001"
    };
```

File: aspnetcore/fundamentals/hosting.md [hunk 2]
```diff
@@ -159,64 +159,64 @@ Use [Configuration](configuration.md) to set configuration values to be used by
...
+        .AddJsonFile("hosting.json", optional: true)
+        .AddCommandLine(args)
+        .Build();
+
+    var host = new WebHostBuilder()
+        .UseConfiguration(config)
+        .UseKestrel()
+        .Configure(app =>
+        {
+            app.Run(async (context) => await context.Response.WriteAsync("Hi!"));
```
Comment (GuardRex) [inline aspnetcore/fundamentals/hosting.md]: Missing one space before "app"

**Comment by Rick-Anderson** [comment]
@GuardRex  Let me know when this is ready to merge.

**Comment by GuardRex** [comment]
@Rick-Anderson Will do ... just waiting on that last spacing update.

File: aspnetcore/fundamentals/hosting.md [hunk 3]
```diff
@@ -159,64 +159,65 @@ Use [Configuration](configuration.md) to set configuration values to be used by
...
-       .Build();
-
-     var host = new WebHostBuilder()
-       .UseConfiguration(config)
-       .UseKestrel()
-       .Configure(app =>
-       {
-         app.Run(async (context) => await context.Response.WriteAsync("Hi!"));
-       })
-     .Build();
```
Comment (GuardRex) [inline aspnetcore/fundamentals/hosting.md]: Make one more little tiny change here: Make sure that The `.Build();` is on the same level as the rest of these. It looks like it needs to go one space to the right.

**Comment by GuardRex** [comment]
@vanillajonathan Yes ... my bad. I found a [@]BillWagner example of the braces with an assignment and a `List<string>` where he justifies them fully under the type. If you can't trust [@]BillWagner when it comes to C#, who the heck can you trust??!! :grinning: We have a few spots in the Docs where it isn't done, so we'll have to massage those into shape as we go. Thanks for fixing these problems.

@Rick-Anderson :shipit:

**Pull request merged**